src/components/views/BlackCardSelection.js

```diff
@@ -122,7 +122,7 @@ const BlackCardSelection = () => {
     // get content
     if(blackCard!==null){
         textContent =
-            <div className={"game description"}>
+            <div className={"game description"} style={{marginTop: "2em", textAlign: "center"}}>
```

Comment:
Can you put the style into the css files?

Reply:
done